question-answering-using-openai/main.bal

```diff
@@ -0,0 +1,153 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/regex;
+import ballerinax/googleapis.sheets;
+import ballerinax/openai;
+
+// Google Sheets API client configuration parameters
+configurable string sheetsAccessToken = ?;
+configurable string sheetId = ?;
+configurable string sheetName = ?;
+
+// OpenAI token
+configurable string openAIToken = ?;
+
+type Request record {|
+    string question;
+|};
+
+type Response record {|
+    string answer;
+|};
```

Comment:
No need of this, use string directly.

Reply:
addressed in [fa5629f](https://github.com/ballerina-guides/ai-samples/pull/2/commits/fa5629f9ca6bfae66689af4f34151a7883b405d2)